sources/packages/backend/apps/db-migrations/src/migrations/1665524525006-CreateInitialMessage.ts

```diff
@@ -0,0 +1,12 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../utilities/sqlLoader";
+
+export class CreateInitialMessage1665524525006 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData("Create-initial-messages.sql", "Messages"),
+    );
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {}
```

Comment:
The method as it is is generating a Sonarcloud issue. Did you try to remove it and see if the revert migration works?

Reply:
IMO create initial messages do not need a separate migration.ts and it can be part of the migration which creates the table so that revert becomes implicit when the table is dropped on the revert.

Reply:
I like the suggestion from @dheepak-aot. @andrewsignori-aot @ann-aot @guru-aot @andrepestana-aot kindly let me know if you have other suggestions.

Reply:
I am ok to have the initial data in the same script 👍